keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/DoubleTensor.java

```diff
@@ -245,6 +247,12 @@ static DoubleTensor placeHolder(int[] shape) {
 
     DoubleTensor standardizeInPlace();
 
+    default public DoubleTensor zero() {
```

Comment:
This should be setAllToZero(). Or can we just use `setAll(0.0)`, which is both shorter and more clear?